api_utils.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <memory>
+#include <fstream>
+#include <iostream>
+#include <ctime>
+#include <fcntl.h> // For ::open
+#include <limits>
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <google/protobuf/text_format.h>
+
+using std::cout;
+using std::endl;
+
+
```

Comment:
Add some namespace?

Reply:
@yinghai It is preferred not to specify a namespace in a .h file. It can be included in <we do not know where> and can mess up things.  .cpp files is a different issue.